profiler/src/test/java/com/splunk/opentelemetry/profiler/EventProcessingChainTest.java

```diff
@@ -88,108 +89,25 @@ void testLifecycle() {
     inOrder.verifyNoMoreInteractions();
   }
 
-  @Test
-  void testFlushOnChunkChange() {
```

Comment:
I guess this is less interesting now that the chunk tracker exists elsewhere? Should we still verify that flush is called somewhere though?

Reply:
I think this is covered with other tests, if `flushBuffer` is not called nothing will get exported with pprof.